### ISSUE TITLE
WAR Bundle can't not have a version

### ISSUE THREAD

**Issue by LukasJaks** [IBM/cics-bundle-maven#224]
It seems like its impossible for the bundle not to have version (as <bundle_id>_<bundle_version>.war). It doesn't seem to help defining "<finalName>${project.artifactId}</finalName>" in "<build>". 

For us, when we deploy the bundle, the URL of the project is the war file name, and as cool it is to have that in development, when we are deploying to "production" the version number just makes the URL complicated.

There seems to be piece of code that no matter what takes the version and appends it to the file name.

![grafik](https://github.com/IBM/cics-bundle-maven/assets/28509989/94c9716d-8442-4376-bb07-0cf23d3c796e)
(cics-bundle-maven-plugin/src/main/java/com/ibm/cics/cbmp/PackageCICSBundleMojo.java static File createCICSBundle())

Would it be possible to add clause that checks if finalName is defined, and if so, use that and don't add version?

**Comment by stewartfrancis** [comment]
Just making sure I understand, you're setting something like:

```xml
<project>
  <build>
    <finalName>helloWorld</finalName>
  </build>
</project>
```

in your Maven pom.xml, and you'd like us to write a CICS bundle called `helloWorld.zip` in the case that `project.build.finalName` is set?

**Comment by LukasJaks** [comment]
Yes, that would be exactly what we need!

**Comment by ind1go** [comment]
@LukasJaks this is available in plugin version 1.0.5.
